src/main.js

```diff
@@ -574,9 +577,21 @@ scene("game", () => {
 
 scene("over", () => {
     add([
-        text("Game over"),
-        pos(10, 10),
+        sprite("game-over"),
+        scale(2.5),
+        pos(30, 0)
     ])
+    add([
+        text("High Score: " + scoreText),
+        pos(gameWidth/2-130, gameHeight/2+90),
```

Comment:
I would just replace this line with these two lines: 

pos(gameWidth / 2, gameHeight / 2 + 90),
anchor("center"),

- then it'll be centered.

Reply:
Changes made as per Rowan, please test after it is merged into the main branch.